snap/snapcraft.yaml

```diff
@@ -24,9 +24,15 @@ description: |
     Dump information about the available Vulkan setup.
   - vkcube (Apache-2.0):
     Test the Vulkan setup.
+  - bandwidthTest (MIT):
+    A sample NVIDIA CUDA application can be used to confirm CUDA operational status.
+    (needs nvidia-core22)
+  - deviceQuery (MIT):
```

Comment:
Should we call these `nvidia-` so it's clear?

Reply:
no preference, i guess we could. Both of those names are widely known / associated with CUDA, in this cammelCase spelling.

Reply:
Right, but if someone comes to g-t-t without CUDA knowledge, they may try them. I'd prefer `nvidia-*`, or maybe even `cuda-*`?